Makefile

```diff
@@ -85,3 +85,11 @@ docker-internal-linkcheck:
 container-internal-linkcheck: link-checker-image-pull
 	$(CONTAINER_RUN) $(CONTAINER_IMAGE) hugo --config config.toml,linkcheck-config.toml --buildFuture
 	$(CONTAINER_ENGINE) run --mount type=bind,source=$(CURDIR),target=/test --rm wjdp/htmltest htmltest
+
+clean-api-reference: ## Clean all directories in API reference directory, preserve _index.md
+	rm -rf content/en/docs/reference/kubernetes-api/*/
```

Comment:
It looks like we may not want to preserve _index.md. TOC fails if directory is not empty.

Reply:
Does this preserve the section `_index` page?

Reply:
What do you mean by TOC? If this is the generator, I've updated it so it does not fail anymore when directory is not empty.

https://github.com/kubernetes-sigs/reference-docs/pull/196

@kbhawkey , yes this command removes all directories but not files, so the _index.md file is preserved.

Reply:
@feloy ok, would we need to update the api-ref-generator submodule to include those most recent changes? It's currently pointed at [this commit](https://github.com/kubernetes-sigs/reference-docs/tree/d06ff1f929f0339d2f7d1ad2cfaa2192df4494c7)

Reply:
@notchairmk You are right, I just updated it, thanks